src/App.tsx

```diff
@@ -1,7 +1,16 @@
-import { Loader } from './components/Loader';
+import cn from 'classnames';
+import {
+  NavLink,
+  Outlet,
+} from 'react-router-dom';
 
 import './App.scss';
 
+const getLinkClass = ({ isActive }: { isActive: boolean }) => cn(
+  'navbar-item',
+  { 'is-active': isActive, 'has-background-grey-lighter': isActive },
```

Comment:
the condition says only about `has-background-grey-lighter`

Reply:
oh, exactly) Thank you 🥰